artman/tasks/publish/local.py

```diff
@@ -73,7 +73,7 @@ def execute(self, gapic_code_dir, git_repo, local_paths,
         dests = []
 
         # Determine where the code belongs and stage it there.
-        for path in git_repo.get('paths', ['.']):
```

Comment:
I am not particularly worried about catching the error as soon as possible in the chain, but I am not opposed to it if it is easy. Definitely change this to a flat error though:

```python
if 'paths' not in git_repo:
    raise RuntimeError('This git repository entry in the artman YAML does not define module paths.')
```

Note that if you do this, you need to go back and add a `.` path to the spots where it is appropriate.